Content.Server/Fluids/EntitySystems/SmokeSystem.cs

```diff
@@ -256,6 +273,25 @@ public void Start(EntityUid uid, SmokeComponent component, Solution solution, fl
         timer.Lifetime = duration;
     }
 
+    private void HandleSmokeTrigger(EntityUid uid, SmokeOnTriggerComponent comp, TriggerEvent args)
+    {
+        var xform = Transform(uid);
+        var smokeEnt = Spawn("Smoke", xform.Coordinates);
```

Comment:
Do not hardcode prototype IDs.

Reply:
SpawnOnTrigger or something definitely exists so this isnt even needed